library/proguard.txt

```diff
@@ -39,6 +39,10 @@
    <methods>;
 }
 
+-keep, includedescriptorclasses class io.envoyproxy.envoymobile.engine.JvmKeyValueStoreContext {
```

Comment:
How is this file updated?

Reply:
Manually, when a new JNI-facing class is added.